doc/appendices/administration.rst

```diff
@@ -0,0 +1,120 @@
+.. Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+.. See the doc/COPYRIGHT file for details.
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _administration:
+
+**************
+Administration
+**************
+
+This appendix includes administrative details describing the roles and
+responsibilities of participants.
+
+
+Community standards
+===================
+
+Be respectful. Communicate openly. Resolve disagreements peacefully (or even
+agree peacefully to postpone their resolution).
+
+
+.. _roles:
+
+Roles
+=====
+
+The roles of the Celeritas code base are related but not identical to the roles
+set out in the SciDAC project management plan (PMP). These roles should be
+reflected in the different teams and/or access restrictions in GitHub (TODO:
+they currently are not).
```

Comment:
Should we just remove the TODO and go ahead with creating those teams?

Reply:
Done.